src/modules/setup/setupTypings.ts

```diff
@@ -97,13 +99,13 @@ const injectTypingsInPackageJson = async (appDeps: Record<string, any>, ignoreLi
   }
 }
 
-export const setupTypings = async (manifest: Manifest, ignoreLinked: boolean, buildersToAddTypes: string[]) => {
+export const setupTypings = async (manifest: Manifest, ignoreLinked: boolean) => {
   const appName = manifest.vendor + '.' + manifest.name
   const appMajor = toMajorRange(manifest.version)
 
   const { builder: builderClient } = createClients({}, { retries: 2, timeout: 2000 })
   const builders = R.keys(R.prop('builders', manifest) || {})
-  const filteredBuilders = R.intersection(builders, buildersToAddTypes)
+  const filteredBuilders = R.intersection(builders, BUILDERS_WITH_TYPES)
```

Comment:
Maybe it can be a constant but I think this could make more difficult to test this function

Reply:
🤔 🤔🤔  thinking  🤔 🤔 🤔

Reply:
agreed!